src/GafferSceneUI/StandardLightVisualiser.cpp

```diff
@@ -451,13 +451,14 @@ Visualisations StandardLightVisualiser::visualise( const IECore::InternedString
 			ConstDataPtr textureData = drawTextured ? surfaceTexture( shaderNetwork, attributes, maxTextureResolution ) : nullptr;
 			boundOrnaments->addChild( const_pointer_cast<IECoreGL::Renderable>( environmentSphereSurface( textureData, maxTextureResolution, color ) ) );
 		}
-		boundOrnaments->addChild( const_pointer_cast<IECoreGL::Renderable>( environmentSphereWireframe( 1.05f, Vec3<bool>( true ) ) ) );
+		boundOrnaments->addChild( const_pointer_cast<IECoreGL::Renderable>( sphereWireframe( 1.05f, Vec3<bool>( true ) ) ) );
 	}
 	else if( type && type->readable() == "spot" )
 	{
 		float innerAngle, outerAngle, radius, lensRadius;
 		spotlightParameters( attributeName, shaderNetwork, innerAngle, outerAngle, radius, lensRadius );
 		boundOrnaments->addChild( const_pointer_cast<IECoreGL::Renderable>( spotlightCone( innerAngle, outerAngle, lensRadius / visualiserScale, 1.0f, 1.0f ) ) );
+		fixedScaleOrnaments->addChild( const_pointer_cast<IECoreGL::Renderable>( sphereWireframe( radius, Vec3<bool>( false, false, true ), 0.5f, V3f( 0.0, 0.0, 0.1 * visualiserScale ) ) ) );
```

Comment:
I'm not a fan of the radius visualisation being moved according to visualiser scale. Doesn't the radius represent the _geometry_ that shadow rays get fired at, and isn't moving that around misleading? I guess the argument is that we need to keep it out of the way of the colour indicator?

Reply:
Its all a bit abstract, for Arnold, its described as:

>  With a non-zero value for radius, then the light source will behave more like a spherical source with the radius specified (although again with the light emitted being constrained to a cone)

In practice, its some kind of hybrid, you can make the radius larger than some object in your scene, and that object still receives shadowing...

Reply:
>  I guess the argument is that we need to keep it out of the way of the colour indicator? 

I think the argument is that unless some part of the visualiser has well-represented world meaning (eg: `lens_radius`, a vis at scale of 1 and 10 should look the same, just bigger.

Reply:
(In this case the offset would not have a world meaning, but the radius of it does)

Reply:
> In practice, its some kind of hybrid, you can make the radius larger than some object in your scene, and that object still receives shadowing...

I'm not sure what this means, but it still seems that the radius is centred at `0, 0, 0` in local space, and not at some arbitrary distance behind it.

Reply:
This was why I did it originally:
![r1s1](https://user-images.githubusercontent.com/896779/74669108-689ca380-519e-11ea-97d2-b58dffbdd6dc.png)
![r10s10wo](https://user-images.githubusercontent.com/896779/74669111-69cdd080-519e-11ea-9693-6552e0a1432a.png)
![r10s10](https://user-images.githubusercontent.com/896779/74669110-69cdd080-519e-11ea-9fbf-5561aa74c055.png)

If you put `radius` vis at `0, 0, 0` then when `lens_radius` == `radius`, as `lens_radius` (which does have absolute world meaning as it represents the visible source of light emission) is already at `0, 0, 0`, you can't see the radius visualisation at all. Daniel attempted to address this by adding chevrons onto the side, which we could add to this, but we generally preferred the simplicity and consistency without.

Reply:
> > In practice, its some kind of hybrid, you can make the radius larger than some object in your scene, and that object still receives shadowing...
> 
> I'm not sure what this means, but it still seems that the radius is centred at `0, 0, 0` in local space, and not at some arbitrary distance behind it.

![image](https://user-images.githubusercontent.com/896779/74669782-c67dbb00-519f-11ea-8088-dc58e503ef4e.png)

The cube is 2 units from the light, of `radius` 5.

Reply:
Other lights, such as the `photometic_light` and `point_light` don't emit inside their radius:

![image](https://user-images.githubusercontent.com/896779/74669948-19f00900-51a0-11ea-8f60-a9f3ad517a92.png) ![image](https://user-images.githubusercontent.com/896779/74670017-3724d780-51a0-11ea-91b3-92544fc260f5.png)

Reply:
How about this as a compromise: https://github.com/GafferHQ/gaffer/pull/3616/commits/ebed8d3886c577226e5f395f057e90e8ca7114a2

Reply:
`<chatted>` and decided to just leave it as the original PR for now and see how it goes.

Reply:
Summary of offline conversation :

- b8e9af5 has the downside that the radius intersects with the colour indicator.
- We think there is a sphere centered at 0 and that shadow rays are thrown at it, but it's confusing when the radius gets big because the shadow rays end up diverging a long way from the light vector.
- I quite like the idea of using a non-offset sphere visualisation, which would still intersect the colour indicator but would be clearer due to having rings in two axes.
- The current offset disk has been run past quite a few people and approved though, so I should let it go. We can revisit later if necessary.

Reply:
> We think there is a sphere centered at 0 and that shadow rays are thrown at it, but it's confusing when the radius gets big because the shadow rays end up diverging a long way from the light vector.

Arnold is inconsistent about whether the source of spotlight is a circle or a sphere, when it comes to the penumbra, some calculations use one, some use the other - this actually causes ugly artifacts in production, which we've been asking them to fix.  Not particularly relevant, except that trying to measure Arnold's behaviour to decide one way or the other.

> I quite like the idea of using a non-offset sphere visualisation, which would still intersect the colour indicator but would be clearer due to having rings in two axes.
If this is refering to Tom's original one with one circle perpendicular to the direction of the light, running straight through the cone, it was pretty unpopular at IE.  Maybe just leave 20 degrees worth of the perpendicular circle where it intersects the plane of the light?  Would that be a form of chevron that would bother Tom less?

> The current offset disk has been run past quite a few people and approved though, so I should let it go. We can revisit later if necessary.

Sounds like the right call.